website/docs/index.html.markdown

```diff
@@ -192,6 +192,30 @@ resource "vcd_network_routed" "net1" {
 
 Note that when connecting with API tokens you can't create or modify users, roles, global roles, or rights bundles.
 
+### Connecting with a Service Account API token
+
+With VCD 10.4.0+, you can connect using a service account API token, as defined in the [documentation](https://blogs.vmware.com/cloudprovider/2022/07/cloud-director-service-accounts.html). Because a new API token is provided on every authentication request, the user is required to provide a file in `json` format with the current API key. e.g:
```

Comment:
Should we state that the file is read+write?

Reply:
Changed.